zebra-chain/Cargo.toml

```diff
@@ -67,7 +67,7 @@ tokio = { version = "1.18.2", optional = true }
 # ZF deps
 ed25519-zebra = "3.0.0"
 # TODO: Update to 0.5 release when published
-redjubjub = { git = "https://github.com/ZcashFoundation/redjubjub.git", rev = "a32ae3fc871bc72558ac2ce7eac933d1ad5f4a9c" }
+redjubjub = { git = "https://github.com/ZcashFoundation/redjubjub.git", rev = "c285d1d381a86d6ae66f3b78b8db278526d76dd8" }
```

Comment:
We should make a release and point to it: https://github.com/ZcashFoundation/redjubjub/pull/147

Reply:
Done in 2baeee3f9